app/controllers/event_log_files_controller.rb

```diff
@@ -44,10 +46,16 @@ def index
     end
 
     begin
+      @select_clause = "SELECT Id, EventType, LogDate, LogFileLength"
+      @order_by_clause = "ORDER BY LogDate DESC, EventType"
+      if @has_one_hr_elf
```

Comment:
If-else is more readable IMO.

if @has_one_hr_elf 
 # hourly ELF
else
 # daily ELF
end